### DISCUSSION TITLE
Welcome to KawaiiLogos Discussions!

### DISCUSSION THREAD

**Discussion by SAWARATSUKI** [SAWARATSUKI/KawaiiLogos#106] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

い　つ　も　の

**Comment by RadhiRasho** [comment]
I'm a software developer and I've been seeing your logos all over the place from React to Vite and so on. 

I really love your style and appreciate all the work you do.

**Comment by alfonsusac** [comment]
@SAWARATSUKI do you have a discord server? sorry for the tag

**Comment by mimmydev** [comment]
はじめまして、さわらつきさん！

めっちゃかわいいテックロゴを作ってくれてありがとうございます！すごく尊敬します。

私はミミーと言います。マレーシアでフロントエンドの開発者をしています。実は、私は今は日本に住んでいないのですが、さわらつきさんのように日本以外の国の人が集まるようなコミュニティに入ってみたいなと思っているんです。おすすめのコミュニティってありますか？

ま、、自分はちょっと内気な性格なんですけど、できれば友達を作れたらいいなって。それに、アニメが好きだし、いつかは日本でキャリアを広げてみたいって思ってるんです。そういう意味でも、コミュニティに参加するのはいい第一歩になるかなって。よかったらアドバイスをいただけると嬉しいです！

これからも応援しますね！！

**Comment by Enzoss100** [comment]
I was there when `/ServiceLogos` was still around with all the other logos (PHP, Rust, Discord, React, Arch Linux, Assembly, Unity, VSCode, etc.) when will those logos be coming back?

**Reply by tpi7** [reply]
I'd like to know as well !

**Comment by eptprofile** [comment]
I confused what is the font family you use to make the logo ;-;

**Reply by alfonsusac** [reply]
yuruka std

**Comment by kir4che** [comment]
Hi! May I use KawaiiLogos to make Instagram Filter?
It's for everyone use but non-commercial.

**Comment by CX330Blake** [comment]
Why is it missing so many things here? Such as Python, GitHub, etc. 🥹🥹🥹
